install-dev/langs/en/data/tab.xml

```diff
@@ -108,7 +108,7 @@
     <tab id="Stock_Coverage" name="Stock Coverage" />
     <tab id="Stock_Management" name="Stock Management" />
     <tab id="Stock_Movement" name="Stock Movement" />
-    <tab id="Stock" name="Stock" />
+    <tab id="Stock" name="Stocks" />
```

Comment:
After interviewing a few native English speakers, it seems the most suitable term is "Stock" in singular form. So let's keep it this way and then it's a matter of translation with Crowdin. In French, it will be translated "Stocks" in plural form.